packages/plugin-hardhat/scripts/test.sh

```diff
@@ -3,5 +3,5 @@
 set -euo pipefail
 
 rimraf .openzeppelin
-buidler compile
+npx hardhat compile
```

Comment:
This shouldn't be necessary, the script should be run via `npm test`.

```suggestion
hardhat compile
```

Reply:
oops, this was a debugging attempt it shouldn't be here!